CMakeLists.txt

```diff
@@ -48,6 +48,10 @@ if(WIN32 OR CYGWIN)
 	add_definitions(-DWIN32_LEAN_AND_MEAN)
 endif()
 
+########################### GENERAL ############################
+set(TOOLS_ARE_BROKEN OFF) # <-- set this flag to ON if we detect any problems, e.g. that user has broken libraries
```

Comment:
We still do support debian 8 on it's default libraries as option so it's needed there,
but cmake warns,
and program warns on start,
and now updated documentation to warn too and recommend debian 9.